python/pyspark/sql/tests/connect/test_parity_functions.py

```diff
@@ -90,8 +90,6 @@ def test_nested_higher_order_function(self):
     def test_np_scalar_input(self):
         super().test_np_scalar_input()
 
-    # TODO(SPARK-41904): Fix nth_value value
```

Comment:
is this `nth_value` issue related?

if so, please also remove `def test_nth_value(self)`

Reply:
OK